From e0005e7a3893746514778b13dc46ab8db3557beb Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@st.com>
Date: Fri, 27 Jan 2017 10:30:01 +0100
Subject: [PATCH 6/7] ARM: dts: STiH407-family: Add missing pwm irq

Add the missing interrupt node to properly probe
the pwm device

Buglink: https://stintbugzilla.st.com/show_bug.cgi?id=108005
Change-Id: Ida754c95a6e51119108e87c3c70665985adf9cfe
Signed-off-by: Lionel Debieve <lionel.debieve@st.com>
Reviewed-on: https://gerrit.st.com/64175
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
Reviewed-by: Patrice CHOTARD <patrice.chotard@st.com>
Tested-by: Patrice CHOTARD <patrice.chotard@st.com>
---
 arch/arm/boot/dts/stih407-family.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/stih407-family.dtsi b/arch/arm/boot/dts/stih407-family.dtsi
index ad68836..11cdd50 100644
--- a/arch/arm/boot/dts/stih407-family.dtsi
+++ b/arch/arm/boot/dts/stih407-family.dtsi
@@ -688,6 +688,7 @@
 			compatible	= "st,sti-pwm";
 			#pwm-cells	= <2>;
 			reg		= <0x9510000 0x68>;
+			interrupts      = <GIC_SPI 131 IRQ_TYPE_NONE>;
 			pinctrl-names	= "default";
 			pinctrl-0	= <&pinctrl_pwm1_chan0_default
 					&pinctrl_pwm1_chan1_default
-- 
2.7.4

